.github/workflows/test.yaml

```diff
@@ -298,6 +298,9 @@ jobs:
       - name: cargo test (debug; ring; -Z minimal-versions)
         run: cargo -Z minimal-versions test --no-default-features --features=ring --locked
 
+      - name: carg test (debug; no provider; -Z minimal-versions)
```

Comment:
s/carg/cargo/

Reply:
This fell away with a better name ("Test no built-in provider build") in a different area of CI away from the minimal versions stuff.